src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,47 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event is an extension of Task.
+ */
+public class Event extends Task{
+    private String at;
+    private String taskType;
+    private LocalDate eventDate;
+
+    /**
+     * Constructor for Event.
+     * @param description Event description.
+     * @param at when the event is happening.
```

Comment:
I believe `when` should be capitalised to `When` as mentioned in the coding standard. Occurs in other places too.